lib/Db/ImageMapper.php

```diff
@@ -199,6 +199,25 @@ public function findImages(string $userId, int $model): array {
 		return $images;
 	}
 
+	/**
+	 * return the first find image as prson's avatar 
+	 */
+	public function getPersonAvatar(Person $person) {
```

Comment:
Don't use it, and remove it..
I understand what you wanted to do, but this is wrong. The avatar is not a photo of a person. If you can't get the face thumnail better show the application icon. :wink: